### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by rektide** [denoland/deno#3839]
Hello. Does Deno support Android? I haven't tried using rust on Android yet, but it'd be lovely to be able to mix in some Deno to an Android app a powerful embedded/scripting runtime. Thought I'd ask. What feats might be required to push this forward, if not?

**Comment by ry** [comment]
Hello @rektide - it has been a long time since I've seen your avatar : )

Deno could support Android, but it's going to be some work. First thing would be to get rusty_v8 to build, somehow passing the target through to GN/Ninja probably the first thing to do would be to try: 
```
GN_ARGS='target_os="android" target_cpu="arm"' cargo build -vv --target aarch64-linux-android
```
Once RustyV8 builds there, it should be pretty easy to port Deno. Does Github Actions support Android arm?

**Comment by gponsinet** [comment]
Hello !
I'm a newbie on rust so every things I will say can be obvious for you.

I tried building it for aarch64-linux-android, so I installed the target with rustup:
`rustup target add aarch64-linux-android`

After running:
`GN_ARGS='target_os="android" target_cpu="arm"' cargo build --verbose --target aarch64-linux-android`
I've had the following error:
```error occurred: Failed to find tool. Is `aarch64-linux-android-clang` installed?```

I search for it with `rustup toolchain list` but I've only one toolchain `stable-x86_64-apple-darwin (default)` and I cannot find way to add another.
Obviously I'm on OSX, maybe other toolchain aren't accessible or maybe I need to have the appropriate arch to build it.

I'm really interested to have a first version running on android and maybe help to share deno on java repositories.
